main/src/main/java/org/mobilitydata/gtfsvalidator/validator/RouteNameValidator.java

```diff
@@ -25,13 +27,23 @@
 /**
  * Validates short and long name for a single route.
  *
+ * <p>References:
+ *
+ * <ul>
+ *   <li>&lt;a href="https://gtfs.org/schedule/reference/#routestxt")"&gt;GTFS routes.txt
+ *       specification&lt;/a&gt;
+ *   <li>&lt;a href="https://gtfs.org/schedule/best-practices/#routestxt"&gt;GTFS routes.txt Best
+ *       Practices&lt;/a&gt;
+ * </ul>
+ *
```

Comment:
This comment block is probably overkill.  If you are looking at `gtfs-validator` source code, you probably know about these resources :)